src/modules/commander/commander_tests/state_machine_helper_test.cpp

```diff
@@ -404,7 +404,7 @@ bool StateMachineHelperTest::mainStateTransitionTest()
 
 		{ "transition: MANUAL to ACRO - not rotary",
 			MTT_ALL_NOT_VALID,
-			commander_state_s::MAIN_STATE_MANUAL, commander_state_s::MAIN_STATE_ACRO, TRANSITION_DENIED },
+			commander_state_s::MAIN_STATE_MANUAL, commander_state_s::MAIN_STATE_ACRO, TRANSITION_CHANGED },
```

Comment:
Done. Feel free to squash these into your commits for cleaner history on master.